pkg/lib/infra/sms/accessyou/apis/sendsms.go

```diff
@@ -0,0 +1,55 @@
+package apis
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/authgear/authgear-sms-gateway/pkg/lib/infra/sms/accessyou/models"
+)
+
+func SendSMS(
+	client *http.Client,
+	baseUrl string,
+	accountNo string,
+	user string,
+	pwd string,
+	sender string,
+	to string,
+	body string,
+) ([]byte, *models.SendSMSResponse, error) {
+	req, _ := http.NewRequest(
+		"POST",
+		fmt.Sprintf(
+			"%v/sendsms.php?accountno=%v&pwd=%v&tid=1&phone=%v&a=%v&user=%v&from=%v",
+			baseUrl,
+			accountNo,
+			pwd,
+			to,
+			body,
+			user,
+			sender),
```

Comment:
You construct URL with url.URL and url.Values, not by string concatenation and printf.